src/ProjectSystemSetup/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "ProjectSystemSetup": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\IDE\\devenv.exe",
```

Comment:
This will not work in multiple scenarios, such as:
* When Community or Professional Edition is installe
* When VS is installed into a non-default path